src/Profiles.ts

```diff
@@ -12,12 +12,10 @@
 import { IProfileLoaded, Logger, CliProfileManager, Imperative, ImperativeConfig, IProfile, Session, ISession } from "@brightside/imperative";
 import * as nls from "vscode-nls";
 import * as os from "os";
-import * as fs from "fs";
 import * as path from "path";
 import { URL } from "url";
 import * as vscode from "vscode";
 import * as zowe from "@brightside/core";
-import * as ProfileLoader from "./ProfileLoader";
```

Comment:
I assume that the profile loader and some other files (i.e. `getDefaultProfile.ts` `getAllProfiles.ts`) can be deleted?

Reply:
Yes they can but I got nervous deleting so much code until we were all comfortable with this solution